Documentation/event-reference.md

```diff
@@ -41,10 +41,25 @@ If the required `xdm` key is not present in the event data payload, the event is
 | --- | ---------- | -------- | ----------- |
 | xdm | <code>Map<String,&nbsp;Object></code> | Yes | XDM formatted data; use an `XDMSchema` implementation for better XDM data ingestion and data format control. |
 | data | <code>Map<String,&nbsp;Object></code> | No | Optional free-form data associated with this event. |
+| config | <code>Map<String,&nbsp;Object></code> | No | Optional config settings. Find the available keys for `config` below.|
 | datasetId | `String` | No | Optional custom dataset ID. If not set, the event uses the default Experience dataset ID set in the datastream configuration. |
-| request | <code>Map<String,&nbsp;Object></code> | No | Optional request parameters. |
+| request | <code>Map<String,&nbsp;Object></code> | No | Optional request parameters. Find the available keys for `request` below. |
 
-> **Note**  
+config
+
+| Key | Value type | Required | Description |
+| --- | ---------- | -------- | ----------- |
+| datastreamIdOverride | `String` | No | Optional Datastream identifier used to override the default datastream identifier set in the Edge configuration. |
+| datastreamConfigOverride | <code>Map<String,&nbsp;Object></code> | No | Optional Datastream configuration used to override individual settings from the default datastream configuration. |
```

Comment:
Use lowercase datastream for these two; "Optional datastream identifier ..." and "Optional datastream configuration ..."